### PR TITLE
6677: Change "Desktop" App to Say "Exit VR"

### PR DESCRIPTION
**Pull request by ElderOrb**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can one of the admins verify this patch?

**Comment by anshuman64** [comment]
add to whitelist

**Comment by ElderOrb** [comment]
for QA:

In HMD mode, open the tablet. The Desktop icon should now say "Exit VR".
In Desktop mode, the toolbar VR icon should now say "Enter VR".

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2240/HighFidelity-Beta-PR11114-58a7af66c2da5bce7cf76659c083349ed18cfe4b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2240/HighFidelity-Beta-PR11114-58a7af66c2da5bce7cf76659c083349ed18cfe4b.exe

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
Testing complete:

In HMD mode, open the tablet. The Desktop icon should now say "Exit VR". **Pass**
In Desktop mode, the toolbar VR icon should now say "Enter VR". **Pass**

File: scripts/system/hmd.js [hunk 1]
```diff
@@ -43,17 +43,20 @@ var tablet = Tablet.getTablet("com.highfidelity.interface.tablet.system");
 // Independent and Entity mode make people sick; disable them in hmd.
 var desktopOnlyViews = ['Independent Mode', 'Entity Mode'];
 
+var switchToVR = "Enter VR"
```
Comment (ZappoMan) [inline scripts/system/hmd.js]: coding standard - line should end with a `;`

File: scripts/system/hmd.js [hunk 2]
```diff
@@ -43,17 +43,20 @@ var tablet = Tablet.getTablet("com.highfidelity.interface.tablet.system");
...
+var switchToDesktop = "Exit VR"
```
Comment (ZappoMan) [inline scripts/system/hmd.js]: coding standard - line should end with a `;`

File: scripts/system/hmd.js (hunk 1 above)
Comment (ElderOrb) [inline scripts/system/hmd.js]: Standard is standard, fixed. :)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2268/HighFidelity-Beta-PR11114-d05e1677c3a58272b14c655bbd940a9d73200edd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2268/HighFidelity-Beta-PR11114-d05e1677c3a58272b14c655bbd940a9d73200edd.exe

**Pull request merged**